lemur/plugins/lemur_aws/plugin.py

```diff
@@ -173,6 +173,77 @@ def get_elb_endpoints_v2(account_number, region, elb_dict):
     return endpoints
 
 
+def get_distribution_dns(distrib_dict):
+    """
+    Build a DNS name for a CloudFront distribution.  Use the default name if no aliases are given,
+    else return the comma-separated aliases.
+    :param distrib_dict:
+    :return:
+    """
+    dns = distrib_dict.get("DomainName")
+    aliases = distrib_dict.get("Aliases")
+    if not aliases or "Items" not in aliases:
+        return dns
+    aliases["Items"].sort()
+    return ','.join(aliases["Items"])
+
+
+def get_distribution_endpoint(account_number, cert_id_to_name, distrib_dict):
+    """
+    Constructs endpoint data from a distribution response, or None if it does
+    not represent a distribution Lemur cares about.
+    :param account_number:
+    :param cert_id_to_name: map of IAM certificate IDs to names
+    :param distrib_dict:
+    :return: a list of endpoint dictionaries
+    """
+
+    cert = distrib_dict["ViewerCertificate"]
+    if not cert:
+        return None
+    # Ignore distributions using the default cert for the cloudfront.net domain
+    if cert.get("CloudFrontDefaultCertificate"):
+        return None
+    # Ignore ACM certificates, since these are auto-rotated
+    if cert.get("ACMCertificateArn"):
+        return None
+
+    iam_cert_id = cert.get("IAMCertificateId")
+    if not iam_cert_id:
+        return None
+
+    cert_name = cert_id_to_name.get(iam_cert_id)
+    if not cert_name:
+        current_app.logger.warning(
+            "get_distribution_endpoints: no IAM certificate with id {0}".format(iam_cert_id))
+        return None
+
+    policy = dict(
+        name='cloudfront-none',
+        ciphers=[]
+    )
+    minimum_version = cert.get("MinimumProtocolVersion")
+    if minimum_version:
+        policy = dict(
+            name='cloudfront-%s' % minimum_version,
```

Comment:
readability

```
name=f'cloudfront-{minimum_version}'
```

Reply:
Missed these comments on the last PR, fixed now.